src/ynEIGEN/TokenStakingNodesManager.sol

```diff
@@ -237,8 +233,10 @@ contract TokenStakingNodesManager is AccessControlUpgradeable, ITokenStakingNode
     /// @notice Sets the maximum number of staking nodes allowed
     /// @param _maxNodeCount The maximum number of staking nodes
     function setMaxNodeCount(uint256 _maxNodeCount) public onlyRole(STAKING_ADMIN_ROLE) {
-        maxNodeCount = _maxNodeCount;
-        emit MaxNodeCountUpdated(_maxNodeCount);
+        if (_maxNodeCount >= nodes.length) {
```

Comment:
instead of silently not setting this i'd rather leave it as 

"won't fix"

i propose we revert this change.

Currently if set lower than current node count it simply prevents creation of new staking nodes anyway